change/@fluentui-react-button-2020-09-02-14-32-55-feat-implementing-v8-conformance-tests.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Adding conformant tests to v8 component test files.",
```

Comment:
For packages where you just changed tests, the change type should be none. (But it should be patch if you added displayNames or fixed other things.)